crates/flox/src/commands/mod.rs

```diff
@@ -435,13 +437,56 @@ impl AdditionalCommands {
             AdditionalCommands::Unsubscribe(args) => args.handle(flox).await?,
             AdditionalCommands::Channels(args) => args.handle(flox)?,
             AdditionalCommands::History(args) => args.handle(flox).await?,
-            AdditionalCommands::PrintDevEnv(args) => args.handle(config, flox).await?,
-            AdditionalCommands::Shell(args) => args.handle(config, flox).await?,
+            AdditionalCommands::PrintDevEnv(args) => {
+                deprecate_command("print-dev-env");
+                args.handle(config, flox).await?
+            },
+            AdditionalCommands::Shell(args) => {
+                deprecate_command("shell");
+                args.handle(config, flox).await?
+            },
```

Comment:
This is printing the deprecation warning a second time

Reply:
oh yeah forgot to remove those -.-

Reply:
done in https://github.com/flox/flox/commit/60d9f11ae7ba41b8a39b0aadb80c4c7947222182